lib/rbui/checkbox/checkbox.rb

```diff
@@ -14,7 +14,7 @@ def default_attrs
         data: {
           rbui__form_field_target: "input",
           rbui__checkbox_group_target: "checkbox",
-          action: "input->rbui--form-field#onInput invalid->rbui--form-field#onInvalid change->rbui--checkbox-group#onChange"
+          action: "change->rbui--checkbox-group#onChange input->rbui--form-field#onInput invalid->rbui--form-field#onInvalid"
```

Comment:
`change->rbui--checkbox-group#onChange` needs to run before the validation of `input->rbui--form-field#onInput` and `invalid->rbui--form-field#onInvalid`